src/Http/Event/ImportEventRequestHandler.php

```diff
@@ -111,6 +112,7 @@ public function handle(ServerRequestInterface $request): ResponseInterface
             new AgeRangeValidatingRequestBodyParser(),
             new CalendarValidatingRequestBodyParser(),
             new BookingInfoValidatingRequestBodyParser(),
+            new PriceInfoValidatingRequestBodyParser(),
```

Comment:
@LucWollants @JonasVHG maybe we should start looking into combining some of these into a class like `OfferValidatingRequestBodyParser` so we don't have to maintain 2 lists of property validators (here + `ImportPlaceRequestHandler`). Can be in another PR later though.

Reply:
Yes, that can certainly help!

Reply:
@JonasVHG this is maybe something that can still be done inside this pull request?

Reply:
@LucWollants , I've tried it but but I'm afraid that it causes too much code changes so I assume that it would be better in a different PR.

Reply:
Done inside https://github.com/cultuurnet/udb3-silex/pull/1014